opt/prepare-aiidalab.sh

```diff
@@ -91,3 +91,6 @@ if [ ! -e /home/${SYSTEM_USER}/apps ]; then
   git checkout ${AIIDALAB_DEFAULT_GIT_BRANCH}
  cd -
 fi
+
+# Activate the right python kernel.
+python -m ipykernel install --user
```

Comment:
can this go in the dockerfile with the other python-setup-related commands?

Reply:
It was originally in the Docker file, but then I moved it here. The reason was this `--user` option. But it is not clear whether the option is at all needed. I will now try it out without `--user`.

Reply:
worked, so I moved it back to the docker file in ac4d402